pkg/config/components/OTelDebugExporter.yaml

```diff
@@ -0,0 +1,27 @@
+name: otel_debug_exporter
+kind: OTelDebugExporter
+summary: Exports pipeline signal traffic to stdout
+description: |
+  Exports signal data in a pipeline to stdout.
+properties:
+  - name: Verbosity
+    summary: The verbosity level of the debug output
+    description: |
+      The verbosity level of the debug output. The default is "basic".
+    type: string
+    validations:
+      - basic
```

Comment:
I'm probably holding these validations wrong. The intent: declare a strict list of valid string values.

Reply:
I don't think validations is implemented yet based on the usage of https://github.com/honeycombio/hpsf/blob/11c6bf27aaae7e6c8f9c7ca5e668cbb875bbf469/pkg/config/templateComponent.go#L45

Reply:
I have changed this to `validations: [nonblank]` like other string-value properties on existing components. I suppose we've got a latent feature here for an `enum` validation or something.